lib/active_model/serializer/reflection.rb

```diff
@@ -109,6 +109,21 @@ def meta(value = nil)
       # @example
       #   has_one :blog do
       #     include_data false
+      #     meta(id: object.blog.id)
+      #     meta liked: object.likes.any?
+      #     link :self do
+      #       href object.blog.id.to_s
+      #       meta(id: object.blog.id)
+      #     end
+      def set_type(value = nil)
```

Comment:
This would be called in a serializer, something like this:
```
belongs_to :commentable do
  set_type object.commentable.type.to_s
end
```

I tried using `type` instead of `set_type` but it wasn't really working. Probably because it is used elsewhere for something else.